spec/features/collection_drafts/forms/required_fields_spec.rb

```diff
@@ -129,6 +129,76 @@
         end
       end
     end
+
+    context 'when checking the accordion headers for required icons' do
+      it 'displays required icons on the Collection Information accordion' do
```

Comment:
The visits should be outside the it blocks.  I think these tests fit in each forms test file, so, spec/features/collection_drafts/forms/collection_information_form_spec.rb, for instance.